src/main/java/net/sf/jabref/logic/logging/GuiAppender.java

```diff
@@ -37,10 +39,25 @@ public static GuiAppender createAppender(@PluginAttribute("name") String name,
         return new GuiAppender(name, filter, layout);
     }
 
-
+    /**
+     * The log event will be forwarded to the {@link LogMessages} archive.
+     * <p/>
+     * All logged messages will pass this method. For each line in the messages of the log event a new log event will be created.
+     * @param event log event
+     */
     @Override
     public void append(LogEvent event) {
-        CACHE.add(new String(this.getLayout().toByteArray(event)));
+        String message = new String(this.getLayout().toByteArray(event));
+        message = message.replace("\n\r", "\n").replace("\r", "\n");
+        // stack traces logged by 'Log.error("message"), e' will be split by new lines so we can create a new log event for each line as 'e.printStackTrace()' would do.
+        if (event.getLevel() == Level.ERROR) {
+            Arrays.asList(message.split("\n")).stream().filter(s -> !s.isEmpty()).forEach(log -> {
+                LogEvent messageWithPriority = Log4jLogEvent.newBuilder().setMessage(new SimpleMessage(log.replace("\n", ""))).setLevel(event.getLevel()).build();
```

Comment:
Is this kind of message-rework really necessary? 
I had the hope that here a simple `LogMessages.getInstance().add(event)` would satisfy and then maybe use `event.getMessage().getFormattedMessage()` in the view (https://github.com/JabRef/jabref/pull/1383/commits/d6c8eb16d9dd0f964d0840c29a615bd19ece3819#diff-d08231c1b1e93f3d52d35f72f3b55821R112).

If this is not possible, please move the code to the view, i.e. determine there how a LogEvent should be displayed.